flink-streaming-java/src/test/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironmentComplexConfigurationTest.java

```diff
@@ -156,4 +190,30 @@ public void onJobExecuted(@Nullable JobExecutionResult jobExecutionResult, @Null
 
 		}
 	}
+
+	/**
+	 * A dummy class to specify a Kryo serializer for.
+	 */
+	public static class CustomPojo {
+	}
+
+	/**
+	 * A dummy Kryo serializer which can be registered.
+	 */
+	public static class CustomPojoSerializer extends Serializer<CustomPojo> {
+		@Override
+		public void write(
+			Kryo kryo,
```

Comment:
I'm doing this just to nitpick, because I think style doesn't matter unless it's enforced:

I think out style guide says to use two indentation levels for parameters. The `.editorconfig` file was recently changed to break this so I suggest to use a good `.editorconfig` file and ignore the one in git using `git update-index --skip-worktree .editorconfig`.